src/w2toolbar.js

```diff
@@ -362,6 +362,14 @@
                 el.html(html);
                 if (it.hidden) { el.css('display', 'none'); } else { el.css('display', ''); }
                 if (it.disabled) { el.addClass('disabled'); } else { el.removeClass('disabled'); }
+                if (it.type == 'check' || it.type == 'drop' || it.type == 'menu') {
```

Comment:
Ah, You're right.. This only seems to fix my bug because I'm selecting the button outside of $(this.box) scope.
In my case, (this) seems to reference an instance that is somehow detached. 
Here's a fiddle: http://jsfiddle.net/mpinkston/fayjfxhe/